source/_integrations/myIO.markdown

```diff
@@ -0,0 +1,52 @@
+---
+title: myIO Server
```

Comment:
The name does not match the manifest file of the integration:

![image](https://user-images.githubusercontent.com/195327/80839737-1c3cfd80-8bfc-11ea-9233-c08e74b6dd1d.png)